x11-wm/qtile/files/qtile.desktop

```diff
@@ -1,6 +0,0 @@
-[Desktop Entry]
```

Comment:
You can't drop this file yet, older versions still need it:

$ grep -rsin "\.desktop"
qtile-0.21.0-r1.ebuild:66:      doins resources/qtile.desktop
qtile-0.21.0-r1.ebuild:69:      doins "${FILESDIR}"/qtile.desktop
qtile-0.17.0-r1.ebuild:44:      doins resources/qtile.desktop
qtile-0.16.1.ebuild:42: doins resources/qtile.desktop

Reply:
The `qtile.desktop` in the files directory is for the wayland backend introduced [v0.18.0](https://github.com/qtile/qtile/blob/v0.18.0/CHANGELOG#L37).

A dedicated file for the wayland backend was made available since [v0.18.1](https://github.com/qtile/qtile/blob/v0.18.1/resources/qtile-wayland.desktop), hence we can patch `qtile-0.21.0-r1.ebuild` to use the desktop file provided by qtile instead of the one provided by the ebuild.